clients/src/main/java/org/apache/kafka/common/errors/OffsetMetadataTooLarge.java

```diff
@@ -19,17 +19,10 @@
 
     private static final long serialVersionUID = 1L;
 
-    public OffsetMetadataTooLarge() {
```

Comment:
I wouldn't expect that client users would be throwing these exceptions themselves. But just in case they are, I will remove this change and others like it. 

@ijuma I assume everything else looks good beyond this single comment? I would prefer to handle all issues/concerns in bulk updates instead of many small iterations on a pr.